e3sm_diags/run.py

```diff
@@ -92,7 +94,7 @@ def run_diags(
         pathlib.Path(prov_dir).mkdir(parents=True, exist_ok=True)
 
         log_dir = os.path.join(prov_dir, LOG_FILENAME)
-        _update_root_logger_filepath_to_prov_dir(log_dir)
+        _add_filehandler(log_dir)
```

Comment:
Now we add a `FileHandler` to the root logger object with the path to `prov` directly.